### ISSUE TITLE
requirements need onnxruntime==1.15.1

### ISSUE THREAD

**Issue by deforum** [collabora/WhisperLive#48]
the new onnxruntime==1.16.0 breaks the whisper server

**Comment by yehiaabdelm** [comment]
Here are the logs with `onnxruntime==1.16.0`. Looks like it breaks while trying to download the VAD model.
```bash
2023-09-27T02:45:15.037255177+02:00 Downloading: "https://github.com/snakers4/silero-vad/archive/master.zip" to /root/.cache/torch/hub/master.zip
2023-09-27T02:45:15.037818730+02:00 Traceback (most recent call last):
2023-09-27T02:45:15.037832275+02:00   File "run_server.py", line 4, in <module>
2023-09-27T02:45:15.037836313+02:00     server = TranscriptionServer()
2023-09-27T02:45:15.037840501+02:00   File "/app/whisper_live/server.py", line 37, in __init__
2023-09-27T02:45:15.037844739+02:00     self.vad_model, _ = torch.hub.load(repo_or_dir='snakers4/silero-vad',
2023-09-27T02:45:15.037849518+02:00   File "/usr/local/lib/python3.8/dist-packages/torch/hub.py", line 399, in load
2023-09-27T02:45:15.037853455+02:00     model = _load_local(repo_or_dir, model, *args, **kwargs)
2023-09-27T02:45:15.037857513+02:00   File "/usr/local/lib/python3.8/dist-packages/torch/hub.py", line 428, in _load_local
2023-09-27T02:45:15.037863564+02:00     model = entry(*args, **kwargs)
2023-09-27T02:45:15.037868153+02:00   File "/root/.cache/torch/hub/snakers4_silero-vad_master/hubconf.py", line 44, in silero_vad
2023-09-27T02:45:15.037872791+02:00     model = OnnxWrapper(os.path.join(model_dir, 'silero_vad.onnx'), force_onnx_cpu)
2023-09-27T02:45:15.037876799+02:00   File "/root/.cache/torch/hub/snakers4_silero-vad_master/utils_vad.py", line 24, in __init__
2023-09-27T02:45:15.037880636+02:00     self.session = onnxruntime.InferenceSession(path, sess_options=opts)
2023-09-27T02:45:15.037884373+02:00   File "/usr/local/lib/python3.8/dist-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 432, in __init__
2023-09-27T02:45:15.037888391+02:00     raise e
2023-09-27T02:45:15.037892368+02:00   File "/usr/local/lib/python3.8/dist-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 419, in __init__
2023-09-27T02:45:15.037910994+02:00     self._create_inference_session(providers, provider_options, disabled_optimizers)
2023-09-27T02:45:15.037915352+02:00   File "/usr/local/lib/python3.8/dist-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 451, in _create_inference_session
2023-09-27T02:45:15.037919940+02:00     raise ValueError(
2023-09-27T02:45:15.037924629+02:00 ValueError: This ORT build has ['AzureExecutionProvider', 'CPUExecutionProvider'] enabled. Since ORT 1.9, you are required to explicitly set the providers parameter when instantiating InferenceSession. For example, onnxruntime.InferenceSession(..., providers=['AzureExecutionProvider', 'CPUExecutionProvider'], ...)
2023-09-27T02:45:28.737931569+02:00 

```

**Comment by coderinthenorth** [comment]
I am getting the same error on my system.
```
Traceback (most recent call last):
  File "/Users/user/Desktop/whisper-server.py", line 2, in <module>
    server = TranscriptionServer()
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/.pyenv/versions/whisper/lib/python3.11/site-packages/whisper_live/server.py", line 32, in __init__
    self.vad_model, _ = torch.hub.load(repo_or_dir='snakers4/silero-vad',
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/.pyenv/versions/whisper/lib/python3.11/site-packages/torch/hub.py", line 558, in load
    model = _load_local(repo_or_dir, model, *args, **kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/.pyenv/versions/whisper/lib/python3.11/site-packages/torch/hub.py", line 587, in _load_local
    model = entry(*args, **kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/.cache/torch/hub/snakers4_silero-vad_master/hubconf.py", line 44, in silero_vad
    model = OnnxWrapper(os.path.join(model_dir, 'silero_vad.onnx'), force_onnx_cpu)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/.cache/torch/hub/snakers4_silero-vad_master/utils_vad.py", line 24, in __init__
    self.session = onnxruntime.InferenceSession(path, sess_options=opts)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/user/.pyenv/versions/whisper/lib/python3.11/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 432, in __init__
    raise e
  File "/Users/user/.pyenv/versions/whisper/lib/python3.11/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 419, in __init__
    self._create_inference_session(providers, provider_options, disabled_optimizers)
  File "/Users/user/.pyenv/versions/whisper/lib/python3.11/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 451, in _create_inference_session
    raise ValueError(
ValueError: This ORT build has ['AzureExecutionProvider', 'CPUExecutionProvider'] enabled. Since ORT 1.9, you are required to explicitly set the providers parameter when instantiating InferenceSession. For example, onnxruntime.InferenceSession(..., providers=['AzureExecutionProvider', 'CPUExecutionProvider'], ...)
```

**Comment by makaveli10** [comment]
https://github.com/collabora/WhisperLive/pull/51

**Comment by makaveli10** [comment]
Closed by #51
